aion/web/elm/src/Socket.elm

```diff
@@ -17,17 +17,22 @@ initializeRoom : Phoenix.Socket.Socket Msg -> String -> ( Phoenix.Socket.Socket
 initializeRoom socket roomIdToString =
     let
         channel =
-            Phoenix.Channel.init ("rooms:" ++ roomIdToString)
+            Phoenix.Channel.init ("room:" ++ roomIdToString)
+
+        roomTopic =
+            "room:" ++ roomIdToString
     in
         Phoenix.Socket.join channel
             (socket
-                |> Phoenix.Socket.on "user:list" ("rooms:" ++ roomIdToString) ReceiveUserList
-                |> Phoenix.Socket.on "new:question" ("rooms:" ++ roomIdToString) ReceiveQuestion
-                |> Phoenix.Socket.on "answer:feedback" ("rooms:" ++ roomIdToString) ReceiveAnswerFeedback
-                |> Phoenix.Socket.on "room:user:joined" ("rooms:" ++ roomIdToString) ReceiveUserJoined
+                |> Phoenix.Socket.on "user_list" roomTopic ReceiveUserList
+                |> Phoenix.Socket.on "answer_feedback" roomTopic ReceiveAnswerFeedback
+                |> Phoenix.Socket.on "user_joined" roomTopic ReceiveUserJoined
+                |> Phoenix.Socket.on "current_question" roomTopic ReceiveQuestion
+                |> Phoenix.Socket.on "display_question" roomTopic ReceiveDisplayQuestion
+                |> Phoenix.Socket.on "question_break" roomTopic ReceiveQuestionBreak
```

Comment:
I think we need to normalise message names. I mean, something like `QUESTION_BREAK, QUESTION_CURRENT, QUESTION_DISPLAY` etc instead of current version which feels really inconsistent and confusing. Thats a separate issue though.

Reply:
Definitely. I think this is where we can start using the correct notation like `question:current` `event:user_joined` etc. I'll refactor that while dealing with #31